rekuho/src/lib/config.ts

```diff
@@ -0,0 +1,35 @@
+import type { Theme } from "./theme"
+import { theme as kanagawa } from "./theme/kanagawa"
+
+type sizeCss = string
+type fontFamilyCss = string
+type fontStyleCss = string
+type pixels = number
+
+export type Font = {
+  family: fontFamilyCss
+  size: sizeCss
+  weight: fontStyleCss
+}
+
+export type Config = {
+  theme: Theme
+  font: Font
+
+  gutterWidth: pixels
+  scrollbarWidth: pixels
+  textOffset: pixels
+}
+
+export const example: Config = {
+  theme: kanagawa,
+  font: {
+    family: "monospace",
+    weight: "normal",
+    size: "20px",
+  },
+
+  gutterWidth: 50,
+  scrollbarWidth: 12,
+  textOffset: 10,
+}
```

Comment:
You really godda stop putting refactors in every commit, lmao
like sure, whatever, let's do it! But this really ought to be a separate commit and PR, ideally :(